hack/common.sh

```diff
@@ -122,10 +122,22 @@ run_WMCO() {
       fi
   fi
 
-  # Add the "openshift.io/cluster-monitoring:"true"" label to the operator namespace to enable monitoring
-  if ! oc label ns $WMCO_DEPLOY_NAMESPACE openshift.io/cluster-monitoring=true --overwrite; then
-      return 1
-  fi
+  # required labels for WMCO namespace
+  declare -a NS_LABELS=(
+    # enable monitoring
+    "openshift.io/cluster-monitoring=true"
+    # turn on the automatic label synchronization required for PodSecurity admission
+    "security.openshift.io/scc.podSecurityLabelSync=true"
+    # set pods security profile to privileged. See https://kubernetes.io/docs/concepts/security/pod-security-admission/#pod-security-levels
+    "pod-security.kubernetes.io/enforce=privileged"
+  )
```

Comment:
WMCO namespace with labels applied:

```
$oc describe ns openshift-windows-machine-config-operator
Name:         openshift-windows-machine-config-operator
Labels:       kubernetes.io/metadata.name=openshift-windows-machine-config-operator
              olm.operatorgroup.uid/04ec388c-2739-4afb-88e4-50c10642787d=
              olm.operatorgroup.uid/573918ed-b56e-4294-bc58-c7d84338c639=
              openshift.io/cluster-monitoring=true
              pod-security.kubernetes.io/enforce=privileged
              security.openshift.io/scc.podSecurityLabelSync=false
Annotations:  openshift.io/sa.scc.mcs: s0:c27,c4
              openshift.io/sa.scc.supplemental-groups: 1000710000/10000
              openshift.io/sa.scc.uid-range: 1000710000/10000
Status:       Active

No resource quota.

No LimitRange resource.
```

Reply:
The openshift-wmco namespace should have `podSecurityLabelSync=true` right? Just want to confirm since the posted output says `false`

Reply:
@saifshaikh48 you are correct. I updated the above output.